bin/openapi

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+declare DOCKER_USER
+DOCKER_USER="$(id -u):$(id -g)"
+
+file=".docs/openapi.json"
+
+# Check if the file exists, if not, create it
+if [ ! -e "$file" ]; then
+    echo "File does not exist, creating..."
+    mkdir -p "$(dirname "$file")" || { echo "Error: Unable to create directory."; exit 1; }
+    touch "$file" || { echo "Error: Unable to create file."; exit 1; }
+    echo "{}" > "$file"  # Initializing with an empty JSON object
+    echo "File created successfully."
+else
+    echo "File already exists."
+    # Empty the file
+    > "$file"
```

Comment:
Would it be a problem if it's not initialized with an empty JSON object?

Reply:
The file is emptied each time the command is run to avoid appending the schema to existing content